incuna-sass/mixins/_incuna-normalise.sass

```diff
@@ -0,0 +1,13 @@
+// Basic styles that we end up adding to all of our projects
```

Comment:
probably american but maybe call incuna-normalize.sass 

this is a good idea :D